tools/build_defs/cmake.bzl

```diff
@@ -19,13 +19,19 @@ load(
 )
 load(":cmake_script.bzl", "create_cmake_script")
 load("//tools/build_defs/shell_toolchain/toolchains:access.bzl", "create_context")
-load("//tools/build_defs/native_tools:tool_access.bzl", "get_cmake_data", "get_ninja_data")
+load(
+    "//tools/build_defs/native_tools:tool_access.bzl",
+    "get_cmake_data",
+    "get_ninja_data",
+    "get_make_data",
+)
 load("@rules_foreign_cc//tools/build_defs:shell_script_helper.bzl", "os_name")
 
 def _cmake_external(ctx):
     cmake_data = get_cmake_data(ctx)
+    make_data = get_make_data(ctx)
 
-    tools_deps = ctx.attr.tools_deps + cmake_data.deps
+    tools_deps = ctx.attr.tools_deps + cmake_data.deps + make_data.deps
```

Comment:
This assumes make is being used which isn't always the case - the ninja generator may be being used.

Reply:
Just noticed this comment today. I'll look into it — ideally we'd know which dependencies to add and only add those.

Reply:
PTAL.